.dockerignore

```diff
@@ -2,3 +2,7 @@
 pkg
 !pkg/sysl_js
 **/node_modules
+/dist/
+/release/
+/pkg/antlr*.jar
+/scrpts/generate/
```

Comment:
`scrpts` is misspelt. `generate` is a file not a directory, so it should not have a trailing slash.